app/frontend/actions.ts

```diff
@@ -37,7 +37,7 @@ import {parseUnsignedTx} from './helpers/cliParser/parser'
 import {TxPlan, unsignedPoolTxToTxPlan} from './wallet/shelley/shelley-transaction-planner'
 import getDonationAddress from './helpers/getDonationAddress'
 import {localStorageVars} from './localStorage'
-import {AccountInfo, Ada, Lovelace, CryptoProviderFeature} from './types'
+import {AccountInfo, Ada, Lovelace, CryptoProviderFeature, _Address, TxType} from './types'
```

Comment:
I would argue that `_` doesn't necessarily mean that it's private, for example in CLI tool we use it as a sign of `internal representation`. I implemented the _Address type a few weeks ago and if so, its not in a scope of this PR to refactor this. If you don't mind we can discuss this in another PR.

Reply:
But I agree that we dont use `_` anywhere else, so we should discuss this for sure.

Reply:
Also for future refactor, we should differentiate between address and string, for example, Address is already validated and a string is just the field-value in the input.

Reply:
It also bugs me whether to name it something like `ValidatedAddress` so that it's extra clear. I didn't know about this distinction and probably while coding, my reaction to a typescript error would be just to change type of my variable from string to Address and move on